site/src/pug/includes/head.pug

```diff
@@ -14,4 +14,7 @@ link(rel="apple-touch-icon" size="128x128" href="/assets/img/logo.png")
 link(rel="stylesheet" href="/assets/css/index.css")
 link(rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cabin")
 
+link(href='https://unpkg.com/mobius1-selectr@latest/dist/selectr.min.css' rel='stylesheet' type='text/css')
+script(src='https://unpkg.com/mobius1-selectr@latest/dist/selectr.min.js' type='text/javascript')
```

Comment:
Although it probably won't be an issue you should probably lock the version

Reply:
Good point!

Reply:
To extend this, it's probably also worth adding [SRI](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to external [css](https://www.w3.org/TR/SRI/#resource-integrity) and js